app/src/main/java/com/greenaddress/greenbits/ui/transactions/TransactionActivity.java

```diff
@@ -105,7 +103,7 @@ protected void onCreate(final Bundle savedInstanceState) {
         } catch (final Exception e) {
             Log.d(TAG, e.getLocalizedMessage());
             finishOnUiThread();
-            return;
+            //return;
```

Comment:
nit: I prefer to remove the line instead commenting it

Reply:
@lvaccaro, yep, I used the comment only to make a double-check with the team, I don't like to make a brute delete of some code without making a double-check the authors